src/data/set/pointwise/smul.lean

```diff
@@ -378,6 +378,15 @@ end vsub
 
 open_locale pointwise
 
+@[to_additive] lemma smul_image [has_smul α β] [has_smul α γ] (f : β → γ) (s : set β) (a : α) :
+  (∀ b, a • f b = f (a • b)) → a • f '' s = f '' (a • s) :=
+image_comm
+
+@[to_additive]
+lemma image_smul' [monoid α] [monoid β] [monoid_hom_class F α β] (f : F) (s : set α) (a : α) :
```

Comment:
What's the name collision here?

Reply:
Ah, it's for the action of `α` on itself. Can you include a comment to that effect?

Reply:
`set.image_smul` is a completely different statement. I could not find any better way of naming them, but arguably this new lemma is less important than the previous. It's analogous to `distrib_mul_action` vs `mul_distrib_mul_action`.

https://github.com/leanprover-community/mathlib/blob/cb3ceec8485239a61ed51d944cb9a95b68c6bafc/src/data/set/pointwise/smul.lean#L155

Reply:
What's? There's no action of `α` on itself in either statement.

Reply:
`a • s` is a pointwise action of `α` on itself. The previous lemma is a pointwise action by `α` on `β`

Reply:
Okay, I think I found a nice naming scheme. `f '' (a • s) = a • f '' s` is `image_smul_comm`. `f '' (a • s) = f a • f '' s` is `image_smul_distrib`.